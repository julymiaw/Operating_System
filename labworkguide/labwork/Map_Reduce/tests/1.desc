测试基本单词计数功能。