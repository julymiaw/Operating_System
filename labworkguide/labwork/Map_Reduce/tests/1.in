A rugby player was hurt very badly during a scrum and he had both of his ears ripped off... Since he was permanently disfigured, he decided to give up playing rugby for good.

 His club and insurance company ensured that a large sum of money went his way.

One day, he decided to invest his money in a small but growing sportswear business. 

He bought the company outright but after signing on the dotted line, realized that he knew nothing about business. 

He decided to employ someone to run the shop.

The next day he set up three interviews. 

The first guy was great. 

He knew everything he needed to and was very enthusiastic. 

At the end of the interview, the former rugby player asked him, “Do you notice anything different about me?”

And the man replied, “Why, yes, I couldn’t help noticing you have no ears.” 

The rugby player got angry and threw him out.

The second interview was with a woman, and she was even better than the first guy. 

He asked her the same question, “Do you notice anything different about me?”

She replied: “Well, you have no ears.” 

He got upset again and showed her the door.

The third and last interview was with the best of the three. 

He was a very young man fresh out of college. 

He was smart and handsome and seemed to know all about the sportswear business.

The rugby player was anxious, but went ahead and asked him the same question: “Do you notice anything different about me?”

To his surprise the young man answered: “Yes, you wear contact lenses.”

The former rugby player was shocked, and said, “What an incredibly observant young man you are. How in the world did you know that?”

The young man fell off his chair laughing hysterically and replied, “Well, it’s pretty hard to wear glasses with no flipping ears!”