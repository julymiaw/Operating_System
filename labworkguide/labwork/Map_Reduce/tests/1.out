A 1
a 5
about 5
after 1
again 1
ahead 1
all 1
an 1
And 1
and 12
angry 1
answered: 1
anxious, 1
anything 3
are. 1
asked 3
At 1
badly 1
best 1
better 1
both 1
bought 1
business. 3
but 3
chair 1
club 1
college. 1
company 2
contact 1
couldn’t 1
day 1
day, 1
decided 3
did 1
different 3
disfigured, 1
“Do 3
door. 1
dotted 1
during 1
ears 1
ears!” 1
ears.” 2
employ 1
end 1
ensured 1
enthusiastic. 1
even 1
everything 1
fell 1
first 2
flipping 1
for 1
former 2
fresh 1
give 1
glasses 1
good. 1
got 2
great. 1
growing 1
guy 1
guy. 1
had 1
handsome 1
hard 1
have 2
he 7
He 7
help 1
her 2
him, 1
him 2
His 1
his 5
How 1
hurt 1
hysterically 1
I 1
in 2
incredibly 1
insurance 1
interview, 1
interview 2
interviews. 1
invest 1
it’s 1
knew 2
know 2
large 1
last 1
laughing 1
lenses.” 1
line, 1
man 5
me?” 3
money 2
needed 1
next 1
no 3
nothing 1
notice 3
noticing 1
observant 1
of 5
off 1
off... 1
on 1
One 1
out 1
out. 1
outright 1
permanently 1
player 5
playing 1
pretty 1
question, 1
question: 1
realized 1
replied: 1
replied, 2
ripped 1
rugby 6
run 1
said, 1
same 2
scrum 1
second 1
seemed 1
set 1
she 1
She 1
shocked, 1
shop. 1
showed 1
signing 1
Since 1
small 1
smart 1
someone 1
sportswear 2
sum 1
surprise 1
than 1
that?” 1
that 2
the 16
The 8
third 1
three 1
three. 1
threw 1
To 1
to 7
up 2
upset 1
very 3
was 11
way. 1
wear 2
“Well, 2
went 2
“What 1
“Why, 1
with 3
woman, 1
world 1
yes, 1
“Yes, 1
you 8
young 4
